### PR TITLE
Add endpoint to create reviews

### PR DESCRIPTION
**Pull request by fabubaker**
Creates an entry for the newly exposed API for creating reviews.

### CODE REVIEW

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      type: text
+      multiline: false
+    - id: pair_f9b5fbc367c148149249ea9d53f02d91
+      name: review[metadata_attributes][review_domain]
+      value: biomedical
+      description: ""
+parameters: []
+headers:
+  - name: Content-Type
+    value: application/x-www-form-urlencoded
```
Comment (hammady) [inline .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml:43]: I believe Rails supports `application/json` out of the box, isn't this how all other endpoints work? Current clients also use this content type by default for all requests.

**Review by hammady — CHANGES_REQUESTED** [review]

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+description: >-
+  This endpoint is used to create reviews given certain parameters such as
+  review title, description and metadata attributes.
+
+
+  Note that parameters are passed as a URL-encoded form.
+method: POST
+body:
+  mimeType: application/x-www-form-urlencoded
+  params:
```
Comment (hammady) [inline .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml:17]: There are more parameters to this request, more metadata and more importantly team id (if user can create team reviews).

Comment (fabubaker) [reply .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml:17]: What exactly are the other parameters?

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml (hunk 1 above)
Comment (fabubaker) [reply .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml:43]: Yeah you're right, I snooped the web request and assumed it had to be url-encoded.

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml (hunk 2 above)
Comment (hammady) [reply .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml]: Basically everything in the metadata object. You can check the models.

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml [hunk 3]
```diff
@@ -13,34 +13,26 @@ description: >-
   Note that parameters are passed as a URL-encoded form.
```
Comment (hammady) [inline .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml:13]: Outdated.

**Review by hammady — CHANGES_REQUESTED** [review]

File: .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml (hunk 2 above)
Comment (fabubaker) [reply .insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml]: I believe I've covered everything?

**Review by hammady — APPROVED** [review]

**Pull request merged**